src/rpc/util.cpp

```diff
@@ -506,6 +513,18 @@ std::string RPCHelpMan::ToString() const
     return ret;
 }
 
+std::string RPCArg::GetFirstName() const
+{
+    auto ret = m_names.substr(0, m_names.find("|"));
+    return ret;
```

Comment:
For style, I believe this could be a one-liner, no? `return m_names.substr(0, m_names.find("|"));` (or have a better variable name than `ret`)

But this is personal taste (and you know the codebase style better than I do :)

Reply:
Thanks, fixed and force pushed.